scvi/inference/inference.py

```diff
@@ -31,13 +34,16 @@ class UnsupervisedTrainer(Trainer):
     """
     default_metrics_to_monitor = ['elbo']
 
-    def __init__(self, model, gene_dataset, train_size=0.8, test_size=None, n_epochs_kl_warmup=400, **kwargs):
+    def __init__(self, model, gene_dataset, train_size=0.8, test_size=0.15, n_epochs_kl_warmup=400, **kwargs):
         super().__init__(model, gene_dataset, **kwargs)
         self.n_epochs_kl_warmup = n_epochs_kl_warmup
         if type(self) is UnsupervisedTrainer:
-            self.train_set, self.test_set = self.train_test(model, gene_dataset, train_size, test_size)
+            self.train_set, self.test_set, self.dev_set = self.train_test(model, gene_dataset, train_size, test_size)
```

Comment:
I think you changed the name of the function to `train_test_dev`

Reply:
yes, thanks for catching.